app/models/interview_question.rb

```diff
@@ -2,4 +2,27 @@ class InterviewQuestion < ActiveRecord::Base
   belongs_to :interview
   validates :interview_id, presence: true
   validates :question_id, presence: true
+  validates :mark, presence: true, on: :update
+
+  def self.get_interview_question_number(interview, interview_question)
+    interview_question_number = 1
+    interview.interview_questions.each_with_index do |q, i|
```

Comment:
could [Array#index](http://ruby-doc.org/core-2.2.0/Array.html#method-i-index) be used here?